### PR TITLE
fix: typo in path fixed and clarification

### PR DESCRIPTION
**Pull request by SimonTheSourcerer**
### Description
Minor documentation fix

### CODE REVIEW

File: .Documentation/articles/innoactive-creator/localization.md [hunk 1]
```diff
@@ -40,3 +40,5 @@ An extension for an english translation would look like this:
 
 When starting the training in Desktop Mode first the EN.json will be loaded and after that the EN_desktop_mode.json which will extend the already existing key/value pairs and overwrite duplicates.
 
+***Be aware that an base file like EN.json always have to exist to make the language selectable.***
```
Comment (Luphoxa) [inline .Documentation/articles/innoactive-creator/localization.md:43]:
```suggestion
***Be aware that a base file like EN.json always has to exist to make the language selectable.***
```

**Review by Luphoxa — CHANGES_REQUESTED** [review]
just small adjustments

**Review by tomwim — APPROVED** [review]
Approved after Irenes fix

**Review by Luphoxa — APPROVED** [review]
Perfect thanks :)

**Pull request merged**